### ISSUE TITLE
custom release-in-Docker step ignored in subproject

### ISSUE THREAD

**Issue by JLofgren** [sbt/sbt-release#264]
I have a multi-project build with a root project that aggregates all the sub-projects. In two of the subprojects I want to publish a *docker* image. 

I have defined this custom release process at the root of `build.sbt`:

```
releaseProcess := Seq[ReleaseStep](
  checkSnapshotDependencies,
  inquireVersions,
  runTest,
  setReleaseVersion,
  commitReleaseVersion,
  tagRelease,
  releaseStepTask(publish in Docker),
  setNextVersion,
  commitNextVersion,
  pushChanges
)
```

However, when I try to release, when it gets to the custom releaseStepTask, it seems to be doing a plain `publish` rather than `publish in Docker`. My evidence is that the process is failing to authenticate to my Artifactory host rather my docker host.  Also, my team has other single-project builds that are successfully using the above custom releaseProcess to publish to Docker.

Is there an obvious problem with how I am doing this? Or is this a bug/non-feature?

Other things I've tried include:

- Defining the `releaseProcess` in each of the sub-project settings
- Defining it in the root project settings
- Defining it as `ThisBuild / releaseProcess`

**Comment by BoopBoopBeepBoop** [comment]
@JLofgren did you ever figure this out? I'm running into the exact same situation

**Comment by JLofgren** [comment]
No, I gave up and split my project into separate repositories in order to work around this.

**Comment by Amos-85** [comment]
@BoopBoopBeepBoop  / @JLofgren ,

if you use the release plugin in the global scope,
you need to declare the subproject like this:
```
releaseStepTask(publish in Docker in subproject1),
releaseStepTask(publish in Docker in subproject2)
```

this way it's worked for me.
